src/xrc/xh_bttn.cpp

```diff
@@ -55,6 +77,17 @@ wxObject *wxButtonXmlHandler::DoCreateResource()
 
     SetupWindow(button);
 
+    SetBitmapIfSpecified(button, &wxBitmapButton::SetBitmapPressed,
```

Comment:
I think this merits a comment saying that we accept the old names here for kind of backwards cross-compatibility with `wxBitmapButton`.